### PR TITLE
Sprint2 messaging

### PR DESCRIPTION
**Pull request by 8BitStone**

### CODE REVIEW

File: src/main/java/ch/bfh/bti7081/Presenter/HomePresenter.java [hunk 1]
```diff
@@ -26,4 +39,35 @@ public HomePresenter(){
...
+    public Doctor getDoctor(ObjectId doctorId){
+        return doctorQuerier.get(doctorId);
+    }
+
+    /**
+     * gets the patient by the Id of one of it's reports
+     * @param reportId
+     * @return
+     */
+    public Patient getPatientByReport(ObjectId reportId){
```
Comment (21adrian1996) [inline src/main/java/ch/bfh/bti7081/Presenter/HomePresenter.java:57]: presenter should not contain filtering. Please move to querier

File: src/main/java/ch/bfh/bti7081/Presenter/MessagePresenter.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    private MessageQuerier messageQuerier;
+
+    public MessagePresenter(){
+        messageQuerier = new MessageQuerier();
+    }
+
+    public List<Message> getMessagesByReportId(ObjectId reportId){
+        // TODO: Sort them by sent date eventually
+        return messageQuerier.getAll()
+                .stream()
```
Comment (21adrian1996) [inline src/main/java/ch/bfh/bti7081/Presenter/MessagePresenter.java]: presenter should not contain filtering. Please move to querier

File: src/main/java/ch/bfh/bti7081/Presenter/HomePresenter.java (hunk 1 above)
Comment (YannisSchmutz) [reply src/main/java/ch/bfh/bti7081/Presenter/HomePresenter.java:57]: Will do

File: src/main/java/ch/bfh/bti7081/Presenter/MessagePresenter.java (hunk 2 above)
Comment (YannisSchmutz) [reply src/main/java/ch/bfh/bti7081/Presenter/MessagePresenter.java]: will do

**Review by YannisSchmutz — APPROVED** [review]
Details-/Objective view is quite buggy at the moment. The location of the bug has been detected though (setViewDetails(createDetailsDrawer(..)) has to be called somewhere else.
Will have a look at it as soon as the other PR's are merged to develop branch.

**Pull request merged**